### PR TITLE
docs: new contribute file

### PR DESCRIPTION
**Pull request by nadavshlezz**
solves #52

### CODE REVIEW

**Review by Mordech — CHANGES_REQUESTED** [review]
Just one comment and I'll approve it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Contributing to DYNAMICO
+
+Welcome, and thank you for your interest in contributing to DYNAMICO!
```
[the remark this answers is not in the document]
Comment (nadavshlezz) [reply CONTRIBUTING.md:3]: sure

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- GitHub issues
+- Any related slack channels or stackoverflow
+
+## Reporting Issues
+
+Have you identified a reproducible problem in Dynamico? Have a feature request? We want to hear about it! Here's how you can make reporting your issue as effective as possible.
+
+### Look For an Existing Issue
+
+Before you create a new issue, please do a search in (https://github.com/Soluto/dynamico/issues) to see if the issue or feature request has already been filed.
```
Comment (Mordech) [inline CONTRIBUTING.md:26]: The link is merged with the word next to it resulting in a broken link. 
Screen Shot 2022-07-10 at 22 50 33
We can solve it by replacing the space character.

And let's make the link more readable:
```suggestion
Before you create a new issue, please do a search in [open issues](https://github.com/Soluto/dynamico/issues) to see if the issue or feature request has already been filed.
```

**Review by Mordech — CHANGES_REQUESTED** [review]

**Review by Mordech — APPROVED** [review]
LGTM. Committed a small fix.

**Pull request merged**